### DISCUSSION TITLE
copilot is not working

### DISCUSSION THREAD

**Discussion by Ritesh-adpl** [community/community#32749] [Copilot Conversations]
hello everyone i am using 60 day free copilot at this time after 10 days it is not working what should i do please suggest any thing

**Comment by iamcjwilliams** [comment]
Hello @Ritesh-adpl  Can you post a screenshot of the error you're receiving?

**Comment by davecheney** [comment]
@Ritesh-adpl 
i'm sorry you are having difficulties. Looking at your telemetry it looks like you are making and receiving completions. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Comment by Ritesh-adpl** [comment]
when ever i was using it was suggesting me some next step but from last two
or three days it's not showing any suggestions what i should do . please
suggest me ASAP.

**Reply by davecheney** [reply]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
